sdk/storage/azure-storage-queue/apireview/azure/storage/queue/aio/queue_client_async.pyi

```diff
@@ -0,0 +1,25 @@
+# Stubs for azure.storage.queue.aio.queue_client_async (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from .._shared.policies_async import ExponentialRetry
+from ..queue_client import QueueClient as QueueClientBase
+from azure.core.async_paging import AsyncItemPaged
+from azure.storage.queue._shared.base_client_async import AsyncStorageAccountHostsMixin
+from azure.storage.queue.models import AccessPolicy, QueueMessage, QueueProperties
+from typing import Any, Dict, List, Optional, Union
+
+class QueueClient(AsyncStorageAccountHostsMixin, QueueClientBase):
```

Comment:
Why timeout deserve to be everywhere and not kwargs? (real question, maybe it's important)

Reply:
timeout will go to kwargs everywhere...like in blobs

Reply:
#7807